dataprofiler/profilers/profile_builder.py

```diff
@@ -997,12 +1001,16 @@ def _clean_data_and_get_base_stats(data, sample_size,
         len_data = len(data)
         if not len_data:
             return data, {
-                "sample_size": 0, "empty_line_count": dict(), "sample": [],
+                "sample_size": 0, "empty_line_count": dict(),
+                "sample": [], "capacity": 0
             }
 
         # ensure all data are of type str
         data = data.apply(str)
 
+        # get capacity
+        base_stats = {"capacity": utils.get_capacity(data)}
+
```

Comment:
Capacity to be replaced with memory_size
readme needs to be updates
and option need to be added to choose databyte size

Reply:
fixed all